docs/resources/alert.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: "fastly"
+page_title: "Fastly: alert"
+sidebar_current: "docs-fastly-resource-alert"
+description: |-
+  Provides a Fastly Alert. Alerts send notifications to custom integrations (e.g., Slack channels, PagerDuty, Microsoft Teams and New Relic) when an observed metric either exceeds or falls below a threshold.
+---
+
+# fastly_alert
+
+Provides a Fastly Alert. Alerts send notifications to custom integrations (e.g., Slack channels, PagerDuty, Microsoft Teams and New Relic) when an observed metric either exceeds or falls below a threshold.
+
+## Example Usage
+
+```terraform
+resource "fastly_service_vcl" "example" {
+  name = "my_vcl_service"
+
+  domain {
+    name = "demo.example.com"
+  }
+
+  force_destroy = true
+}
+
+resource "fastly_alert" "example" {
+  name = "my_vcl_service error rate"
+  service_id = fastly_service_vcl.example.id
+  source = "stats"
+  metric = "status_5xx_rate"
+
+  evaluation_strategy {
+    type = "above_threshold"
+    period = "5m"
+    threshold = 0.1
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `evaluation_strategy` (Block List, Min: 1, Max: 1) Evaluation strategy for the Alert. (see [below for nested schema](#nestedblock--evaluation_strategy))
+- `metric` (String) The name of the metric to monitor for Alert evaluation. For detailed information on metrics available for monitoring, see [Fastly Alerts API Documentation](https://developer.fastly.com/reference/api/observability/alerts/definition/).
+- `name` (String) Summary text of the Alert.
+- `service_id` (String) ID of the Fastly Service that the Alert monitors.
+- `source` (String) The metric source. One of: `stats`, `origins`, `domains`.
+
+### Optional
+
+- `description` (String) Text body included in Alert notifications.
+- `dimensions` (Block List, Max: 1) Dimension filters applied to the metric. (see [below for nested schema](#nestedblock--dimensions))
+- `integration_ids` (Set of String) Set of IDs of integrations that Alert notifications will be sent to.
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
```

Comment:
@darin-nee There is also `object` = "definition" that is returned in the response, also read-only

Reply:
Yep, that's in the API response and elided in the Terraform schema.